tuf/ngclient/updater.py

```diff
@@ -129,6 +130,23 @@ def refresh(self) -> None:
             DownloadError: Download of a metadata file failed in some way
         """
 
+        if self.config.lazy_refresh:
+            try:
+                # Try loading only local data
+                data = self._load_local_metadata(Timestamp.type)
+                self._trusted_set.update_timestamp(data)
+                data = self._load_local_metadata(Snapshot.type)
+                self._trusted_set.update_snapshot(data, trusted=True)
+                data = self._load_local_metadata(Targets.type)
+                self._trusted_set.update_delegated_targets(
+                    data, Targets.type, Root.type
+                )
+                return
+            except (OSError, exceptions.RepositoryError):
```

Comment:
Does it make sense to have a `DEBUG` or similar log statement on this error case? I assume what it means is that a client that has configured `lazy_refresh` will be making network request(s) anyways, which would be useful to surface in downstream logging if errors arise 🙂